package.json

```diff
@@ -9,7 +9,7 @@
   },
   "esy": {
     "build": "dune build -p App",
-    "buildDev": "refmterr dune build -p App",
+    "buildDev": "dune build -p App",
```

Comment:
If we want to keep `refmterr` around we can look to [contributing upstream](https://github.com/reasonml/reason-native/blob/aec0ac681be7211b4d092262281689c46deb63e1/refmterr.json#L12) whether ocaml 4.12 can be supported.